src/modules/postComment.js

```diff
@@ -0,0 +1,14 @@
+export default ({ id, username, comment }) => {
```

Comment:
It would be better if you define it as a single object like `payload` for clarity of your code